src/transformers/modeling_flax_pytorch_utils.py

```diff
@@ -88,6 +88,12 @@ def is_key_or_prefix_key_in_dict(key: Tuple[str]) -> bool:
         pt_tensor = pt_tensor.transpose(2, 3, 1, 0)
         return renamed_pt_tuple_key, pt_tensor
 
+    # conv1d layer
```

Comment:
As stated in https://github.com/huggingface/transformers/pull/15519#discussion_r800514504 I'm pretty sure we actually don't need this change. Could you remove it here and improve the comments as mentioned in the link? :-)